site/content/en/docs/Prerequisite Knowledge/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Prerequisite Knowledge"
+linkTitle: "Prerequisite Knowledge"
+weight: 3
+description: >
+  Foundational knowledge you should know before starting working with Agones.
+---
+
+Agones is built on top of the foundation of multiple open source projects, as well as utilising
+several architectural patterns across both distributed and multiplayer game systems -- which can
+make it complicated to get started with, if they are things you are not familiar with or have
+experience with already.
+
+To make getting started easier to break down and digest, this guide was written to outline what concepts and
+technology that the documentation assumes that you have knowledge of, and the
+depth of that knowledge, as well as providing resource to help fill those knowledge gaps.
+
+## Docker and Containerisation
+
+Docker and containerisation is the technological foundation of Agones, so if you aren't familiar,
+we recommend you have knowledge in the following areas before getting started with Agones:
+
+* Containers as a concept
+* Running Docker containers
+* Building your own container
+* Registries as a concept
+* Pushing and pulling containers from a registry
+
+### Resources
+
+The following resources are great for learning these concepts:
+
+* [Docker Overview](https://docs.docker.com/get-started/overview/)
+* [Docker "Quickstart" guide](https://docs.docker.com/get-started/)
+* [Katacoda's free, interactive Docker course](https://www.katacoda.com/courses/docker)
+
+## Kubernertes
+
+Kubernetes builds on top of Docker to run containers at scale, on lots of machines. 
+If you have yet to learn about Kubernetes, we recommend that you have knowledge in the following
+areas before getting started with Agones:
+
+* Kubernetes as a concept
+* Pods
+* Deployments
+* Services
+* Creating a Deployment with a Service
+
+### Resources
+
+* [You should totally read this comic, and interactive tutorial](https://cloud.google.com/kubernetes-engine/kubernetes-comic/)
+* [Kubernetes basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+* [Katacoda's free, interactive Kubernetes course](https://www.katacoda.com/courses/kubernetes)
+* [Kubernetes concepts, explained](https://kubernetes.io/docs/concepts/)
+
+## Game Engine
+
+If you are building a multiplayer game, having some knowledge of a game engine is a requirement. There are multiple
+possible solutions, but the engines that have out of the box SDK's for Agones are:
+
+* [Unity](https://unity.com/)
+* [Unreal](https://www.unrealengine.com/)
+
```

Comment:
I wasn't quite sure whether to link to engines here as well -- is that being an advocate of them over other engines?

But at the same time, if you've never used some kind of game engine they are probably the best place to learn about building a game in an engine.

We could link to the toy game engine/simple-game-server  -- but does that let people learn about game engines? Probably not (which is kind of the point of this section).

..then I also wondered if we should point to multiplayer networking in a game engine (Unity might be more complicate than Unreal since most people I know doing production multiplayer Unity aren't using anything from Unity - they are using OSS or commerical tools).

So.... what do you think of that?

Reply:
If you wanted to be impartial about game engines you could always link to something like https://en.wikipedia.org/wiki/List_of_game_engines. 
I think it is beneficial to know which engines are tried and tested though.

Reply:
You know - this makes me question whether we should have engines here at all. 

@roberthbailey your point makes a lot of sense. You don't actually NEED to know how an engine works to use Agones. There's no point in learning how to do animations in Unity for example if you're only ever going to be working server side moving RPCs around.

Maybe this is an optional section? Or maybe even drop it at all? 

Or maybe this should lean us more into fleshing out the "Dedicated Game Servers" section, and see what guides we can find for supported engines with SDKs?

Reply:
I think it would be enough to change "If you are building a multiplayer game, having some knowledge of a game engine is a requirement" to "If you are building a multiplayer game, you will eventually need to understand how your [game engine](https://en.wikipedia.org/wiki/Game_engine) will integrate with Agones" (leaving the rest of the section alone) and also moving this section below "Dedicated Game Servers" which I think is more important. It wouldn't hurt to flush out the "Dedicated Game Servers" section more as well, but you already have some helpful links there so I don't think that should block this PR.

Reply:
I like it. Done.

I also moved it below dedicated game servers, since it seemed to me that we should have the required stuff before we covered the more "nice to have eventually".